app/controllers/admin/application_controller.rb

```diff
@@ -5,18 +5,18 @@ class ApplicationController < ApplicationController
     after_action :verify_authorized
 
     HELP_URLS = {
-      badges: "https://forem.gitbook.io/forem-admin-guide/admin/badges",
-      badge_achievements: "https://forem.gitbook.io/forem-admin-guide/admin/badges",
-      configs: "https://forem.gitbook.io/forem-admin-guide/admin/config",
-      navigation_links: "https://forem.gitbook.io/forem-admin-guide/admin/navigation-links",
-      pages: "https://forem.gitbook.io/forem-admin-guide/admin/pages",
-      podcasts: "https://forem.gitbook.io/forem-admin-guide/admin/podcasts",
-      reports: "https://forem.gitbook.io/forem-admin-guide/admin/reports",
-      users: "https://forem.gitbook.io/forem-admin-guide/admin/users",
-      html_variants: "https://forem.gitbook.io/forem-admin-guide/admin/html-variants",
-      display_ads: "https://forem.gitbook.io/forem-admin-guide/admin/display-ads",
-      chat_channels: "https://forem.gitbook.io/forem-admin-guide/admin/chat-channels",
-      tags: "https://forem.gitbook.io/forem-admin-guide/admin/tags"
+      badges: "https://admin.forem.com/admin/badges",
```

Comment:
https://admin.forem.com/admin/badges is a 404 for me?  As is https://admin.forem.com/badges without the /admin/ path component.

Reply:
I see - /docs/quick-start-guide/ and /docs/admin/badges!

Reply:
so I double checked the new values from the last commit and I get a 404 for d/a/users and d/a/config only now.


```shell
urls=("https://admin.forem.com/docs/admin/badges"
 "https://admin.forem.com/docs/admin/badges"
 "https://admin.forem.com/docs/admin/config"
 "https://admin.forem.com/docs/admin/navigation-links"
 "https://admin.forem.com/docs/admin/pages"
 "https://admin.forem.com/docs/admin/podcasts"
 "https://admin.forem.com/docs/admin/reports"
 "https://admin.forem.com/docs/admin/users"
 "https://admin.forem.com/docs/admin/html-variants"
 "https://admin.forem.com/docs/admin/display-ads"
 "https://admin.forem.com/docs/admin/chat-channels"
 "https://admin.forem.com/docs/admin/tags")

for url in ${urls[@]} do
  echo "$url" $(curl -I -s "$url" | grep HTTP)
done

https://admin.forem.com/docs/admin/badges HTTP/2 301 
https://admin.forem.com/docs/admin/badges HTTP/2 301 
https://admin.forem.com/docs/admin/config HTTP/2 404 
https://admin.forem.com/docs/admin/navigation-links HTTP/2 301 
https://admin.forem.com/docs/admin/pages HTTP/2 301 
https://admin.forem.com/docs/admin/podcasts HTTP/2 301 
https://admin.forem.com/docs/admin/reports HTTP/2 301 
https://admin.forem.com/docs/admin/users HTTP/2 404 
https://admin.forem.com/docs/admin/html-variants HTTP/2 301 
https://admin.forem.com/docs/admin/display-ads HTTP/2 301 
https://admin.forem.com/docs/admin/chat-channels HTTP/2 301 
https://admin.forem.com/docs/admin/tags HTTP/2 301 
```

Reply:
Fixed, thanks for being more thorough than me 😅

Reply:
https://admin.forem.com/docs/admin/config/README 
https://admin.forem.com/docs/admin/users/README

Looks like topics with child pages land on path/README and don't automatically redirect from path/ to path/README?